flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/PeriodicMaterializationManager.java

```diff
@@ -96,6 +102,11 @@
                 Executors.newSingleThreadScheduledExecutor(
                         new ExecutorThreadFactory(
                                 "periodic-materialization-scheduler-" + subtaskName));
+
+        this.initialDelay =
+                // randomize initial delay to avoid thundering herd problem
+                MathUtils.murmurHash(Objects.hash(operatorId, subtaskIndex))
```

Comment:
The `operatorId` which comes from `OperatorSubtaskDescriptionText` already contains the information of `subtaskIndex`, why we still need another `subtaskIndex` here?

Reply:
Good point, I'll remove `subtaskIndex`.